### DISCUSSION TITLE
Sobre ser PJ e contribuir com o INSS

### DISCUSSION THREAD

**Discussion by dionarodrigues** [frontendbr/forum#471] [Discussão]
É muito comum, atualmente, desenvolvedores front-end trabalhar como PJ. No entanto, para quem trabalha - ou já trabalhou nessa modalidade - sabe que temos que lidar com todos os gastos que um CLT poderia não ter: planos de saúde e odontológico, transporte, refeições, contadores... e INSS.

Sobre esse último item, o INSS, como vocês que são (ou foram) PJs costumam lidar?

- Precisa de algum cadastro na previdência social?
- Como vocês pagam ele?
- Pagam o referente à quantos salários mínimos?
- Qual código vocês utilizam?

Essas minhas dúvidas (e talvez de outras pessoas) surgiram do fato de eu não conseguir uma explicação clara dos funcionários da previdência, contadores ou pesquisando na internet.

**Comment by fdaciuk** [comment]
Fala @diogorodrigues! Sobre o INSS, ele é obrigatório (infelizmente) para PJ quando você retira pró-labore. O valor mínimo para pagamento é 11% sobre o salário mínimo, mas você pode pagar mais se quiser.

Não vejo muita vantagem em pagar INSS, por isso o **infelizmente** ali em cima xD
Se você fizer as contas, ao invés de pagar INSS hoje para receber um salário mínimo quando se aposentar, é muito mais válido **investir** esse dinheiro, em renda fixa, por exemplo.

Por não ter muito o que fazer, pois o INSS é obrigatório, eu procuro pagar o mínimo mesmo, e investir uma grana por mês que servirá de aposentadoria no futuro. Pra ser sincero, comecei a fazer isso esse ano. Ano passado eu li, assisti e estudei muito sobre investimentos. No youtube tem ótimos canais falando sobre isso, principalmente sobre Tesouro Direto.

Fazendo esse tipo de investimento, quando você estiver na idade de se aposentar, vai ter muito mais dinheiro do que se tivesse apenas pagando INSS, e poderá se organizar para gastá-lo da forma que lhe for mais útil =)

Um outro ponto que eu também acho importante comentar sobre o INSS: o valor que você paga hoje vai servir para pagar as pessoas que já estão aposentadas, ou seja: esse seu dinheiro não vai ficar guardado, rendendo.

Isso significa que, para receber hoje, os aposentados pagaram por muito tempo INSS para outros aposentados.

Entendendo essa lógica, e vendo que, hoje em dia a maior parte da galera mais jovem já está abrindo a mente para essas questões, existe uma grande probabilidade de quando você for se aposentar, talvez nem poder receber o valor da sua aposentadoria, pois é a próxima geração que vai ter que pagar INSS pra te sustentar. Quem garante que as leis irão continuar e essa galera vai querer pagar INSS ao invés de investir em si próprios?

Por isso eu penso que um investimento em renda fixa (talvez até no tesouro pré-fixado, que você sabe o valor que vai ter no final) valha muito mais a pena do que depender do INSS =)

**Comment by andrewsfg** [comment]
Eu tenho uma dúvida em relação a isso. Como faz pra ter e pagar sendo PJ? Tem que ir todo mês pagar carnê?

Valeus!

**Comment by fdaciuk** [comment]
@andrewsfg nesse caso você precisa pagar uma guia chamada GPS. Normalmente a sua contabilidade te passaria essa informação, mas se for fazer por conta própria, pode acessar [esse link](http://idg.receita.fazenda.gov.br/orientacao/tributaria/pagamentos-e-parcelamentos/emissao-e-pagamento-de-darf-das-gps-e-dae/gps-guia-da-previdencia-social-orientacoes-1) que lá tem mais informações de como gerar as guias para pagamento =)

**Comment by andresantos-tech** [comment]
No caso de MEI a contribuição fica no mesmo carnê DAS pago mensalmente. Em outras modalidades optantes pelo Simples deve ser da mesma maneira.

Apenas em complemento ao que o @fdaciuk disse, é importante lembrar que *seguridade social não é só aposentadoria* (inclusive, acho que aposentadoria é a última coisa em que se deve pensar, afinal quem é mais jovem só terá direito ao completar 70 anos, e isso se a lei não mudar). Licença maternidade, auxilio doença, pensão por invalidez e programas de renda mínima também vêm da previdência social, por exemplo.

Eu acho que é válido pagar o mínimo como prevenção para esses casos (torcendo é claro pra nunca precisar). Em condições normais, sem imprevistos, o mais "garantido" (com muitas aspas, afinal garantia não existe) para aposentadoria e seguir as colocações do @fdaciuk.

**Comment by andrewsfg** [comment]
Entendi. Vou ver com a minha contadora. Muito obrigado pela resposta @andre-forweb e @fdaciuk.

**Comment by dionarodrigues** [comment]
Ótima estratégia @fdaciuk  Já tem bastante tempo que penso em investir no tesouro direto também, pelos benefícios que vc mencionou realmente é bem vantajoso.

Sobre a previdência, realmente acho algo muito confuso.  Você paga o INSS através da PJ ou PF?

Minha contadora só me envia a guia do GPS pra eu pagar. Reparei que antes ela mandava às vezes. Questionei ela hoje e ela me mandará todo mês. Pelo o que eu entendi, recolhendo esse GPS não precisa pagar INSS como pessoa física.

**Comment by felipefialho** [comment]
@fdaciuk Totalmente de acordo com sua estratégia ;)

**Comment by andrewsfg** [comment]
Eu atualmente invisto no **Tesouro Direto**, mas fico preocupado em não contribuir com nada no INSS, já que posso voltar a ser CLT (vai saber né rs). Mas nem é muito pela aposentadoria (que ficou impossível), é mais pela questão do auxílio doença, ou qualquer imprevisto que eu possa precisar de algum benefício no futuro.

Mas me parece tão burocrático...

**Comment by bertelledani** [comment]
Super de acordo com a estratégia do @fdaciuk! Vou falar um pouco da minha:

Atuando como PJ é importante lembrar que não existe FGTS, nem 40% de multa em rescisão de contrato, nem aposentadoria... o que particularmente acho fantástico pois geralmente você recebe mais que um CLT e tem total liberdade para investir. Fica minha recomendação de todo mês manter a disciplina de separar uma porcentagem para investimentos, de preferência com um fundo de reserva para emergências que tenha liquidez diária (recomendo Sofisa Direto 100% do CDI) e algo de médio e longo prazo. Tesouro é bem legal (juros composto ❤️ ), mas deem uma olhada em CDB's e debêntures também, é importante ter uma carteira de investimentos diversificada e não deixar tudo investido no mesmo lugar.

Estudem e pesquisem bastante sobre investimentos para perder o medo, experimentem nos simuladores. O site Me Poupe da Nathalia Arcuri é bem legal: http://mepoupenaweb.uol.com.br/simuladores-online-de-investimentos/

Sobre a parte de imprevistos @andrewsfg, além dessa reserva de emergência com liquidez diária que eu indiquei, você também pode contratar um seguro de acidentes pessoais que tenha cobertura por "incapacidade temporária". Esse tipo de seguro é comum e o foco são justamente os profissionais autônomos/liberais. O valor desse seguro normalmente está ligado ao valor que você gostaria de receber nesta "incapacidade temporária". Fiz um no meu banco mesmo, fico tranquila que todo mês tem o débito na conta e estou protegida.

Tanto na reserva com liquidez diária quanto na escolha do seguro, se você acha que é suficiente garantir seu sustento por 6 meses e tem uma média de gastos de R$ 2.000 por mês, um seguro com cobertura e/ou reserva de emergência de R$ 12.000 é suficiente. Sacou? 😄

**Comment by lagden** [comment]
nem férias...
e compre hbor3.... LP
allin... rendimento de 50% a 100% em de 6 meses a um ano

**Comment by andrewsfg** [comment]
@bertelledani obrigado pela dicas valiosas. Atualmente invisto no Tesouro Direto, uma parte na Selic (para curto prazo) e outra no IPCA+, longo prazo. Eu gosto bastante do canal Me Poupe, foi lá que entendi um pouco sobre investimentos e tirei várias dúvidas. Estou engatinhando para investimentos mais complexos.

Sobre seguro, é uma boa ideia. Eu tenho do Youse, mas preciso ver essa questão de incapacidade temporária, acredito que meu seguro é só de vida mesmo. 

Abração!

**Comment by oguilhermelessa** [comment]
@fdaciuk Concordo completamente. E falando sobre investimento para o futuro, lançaram uma boa ferramenta chamada **[OiWarren](https://oiwarren.com)**. Ela auxilia você na parte de gestão de investimento próprio, inclusive ela te ajuda comprando pequenas ações no mercado, podendo ter acréscimos no valor. Muito legal!

**Comment by brunotdantas** [comment]
@diogorodrigues muito legal sua thread cara, tirou várias dúvidas, no exercício de simples, até então eu sempre tirei o mínimo de pró-labore (pagando o mínimo de INSS) e declarando no IR a participação dos lucros da empresa. 
Porém seguindo desta forma pagava 15% de imposto (DAS) na nota :/. Descobri então o funcionamento do [Fator R](https://conube.com.br/blog/fator-r-simples-nacional/) e passei a pagar menos imposto porém declarar IR, no custo benefício, vale muito a pena.

**Comment by dionarodrigues** [comment]
Show @brunotdantas . Por conta de um intercâmbio, precisei encerrar a minha empresa antiga e quando abri uma nova entrei nessa técnica do fator R também.

Para os que ainda não estão a par, hoje como todas as atividades de TI se enquadram em no mínimo 15% de impostos, existe uma maneira de pagar apenas 6% de modo legal. O sócio precisa ter um pró-labore de no mínimo 28% sobre os redimentos do mês (ou um salário mínimo caso não haja rendimentos) durante os últimos 12 meses. Com isso, os impostos caem para 6%. Para quem já tem empresa, ou começa a fazer isso desde já e aguarda um ano para ter essa vantagem do fator R ou verifica se não vale mais a pena fechar a empresa atual e abrir outra (e fazendo esse esquema desde o início, passa a pagar 6% desde a abertura). 

Com isso, o INSS passa a ser 11% sobre o pró-labore.

----

Essa nova empresa eu abir com a Contabilizei. O que não estou curtindo é o fato da Contabilizei emitir um imposto de renda como pessoa jurídica e depois eu ter que emitir outro como pessoa física, uma vez que antes eu só emitia como pessoa física, declarando a minha empresa e meus rendimentos juntos. Mas isso eu ainda preciso pesquisar melhor sobre como funciona. Alguém sabe me dizer se antes eu fazia errado, se as regras mudaram ou se eu não preciso do IRPJ uma vez que posso fazer junto com o da PF?

**Comment by fdaciuk** [comment]
> Essa nova empresa eu abir com a Contabilizei. O que não estou curtindo é o fato da Contabilizei emitir um imposto de renda como pessoa jurídica e depois eu ter que emitir outro como pessoa física, uma vez que antes eu só emitia como pessoa física, declarando a minha empresa e meus rendimentos juntos. Mas isso eu ainda preciso pesquisar melhor sobre como funciona. Alguém sabe me dizer se antes eu fazia errado, se as regras mudaram ou se eu não preciso do IRPJ uma vez que posso fazer junto com o da PF?

Como MEI não é necessário o IRPJ, pois você só faz uma declaração anual simples no Portal do Empreendedor, e pode declarar normalmente só o seu IRPF. 

Já como ME "pra cima", o IRPJ é obrigatório, mas quem faz isso é a sua contabilidade.

Você pode continuar declarando normalmente o seu IRPF, só que dessa vez, a contabilidade vai te disponibilizar um informe de rendimentos pra você declarar no seu IRPF.

Esse informe é o mesmo documento que você vai solicitar para bancos onde você tem conta, corretoras de investimentos, etc. Basicamente todos os lugares onde você tenha entrada de rendimentos tributáveis (lucro, no caso, não é tributável).

Se todos os seus rendimentos tributáveis somados do ano estiverem abaixo do mínimo obrigatório para a declaração (tem que pesquisar o valor exato, não lembro de cabeça), não precisa se preocupar em declarar o IRPF.
